### ISSUE TITLE
Statement Insertion Error

### ISSUE THREAD

**Issue by samjaved** [LearningLocker/learninglocker#1199]
Hello,
I am currenly working on my thesis in which i need to use some LRS.So  Learning Locker was my best choice.I installed it on my local machine and started the ui and api server.Now i am trying to insert the statement into LRS  according to the documentation following is the format of my statement:

POST http://www.example.org/data/xAPI/statements
Authorization: YOUR_BASIC_AUTH
X-Experience-API-Version: 1.0.3
Content-Type: application/json; charset=utf-8

[{
  "id": "dfb7218c-0fc9-4dfc-9524-d497097de027",
  "actor": { "mbox": "mailto:test1@example.org" },
  "verb": { "id": "http://www.example.org/verb" },
  "object": { "id": "http://www.example.org/activity" },
}, {
  "actor": { "mbox": "mailto:test2@example.org" },
  "verb": { "id": "http://www.example.org/verb" },
  "object": { "id": "http://www.example.org/activity" },
}]

I am getting response 200 back but it doesn't create any statement in the database and the reponse body shows this error:

RESTRICT_CREATE_ORGANIZATION

Can someone please help me to sort out this issue.

**Comment by EmmanVazz** [comment]
How did you resolve this issue? Thanks

**Comment by samjaved** [comment]
I was sending statements to the URL.As learning locker containts three services(worker,Ui,api) in Learning locker project and one service in xapi project.you need to send the statements to service of xapi project which normally runs with default configuartion on 8081 port.

**Comment by EmmanVazz** [comment]
So don't use the xAPI endpoint the UI shows. The localhost:3000/data/xAPI endpoint?

**Comment by samjaved** [comment]
Yes don’t use this.if you have cloned the xapi project as mentioned in learning locker documentation and you have started the service then you can send the statements to localhost:8081/data/xAPI

**Comment by samjaved** [comment]
Ok.logged in to your UI using locahost:3000 and then go to the clients option there you will get the BASIC AUTH key.

**Comment by EmmanVazz** [comment]
Ah fixed my issue. I needed to create a store to get the proper keys. I was just creating a new client and those keys gave me the same issue. Once I created a store, a new client was created and those keys worked. Thanks

**Comment by shahabuddin11** [comment]
i have given correct Basic_AUTH_KEY ans key and secret also but getting unauthorised access.Can please tll how xapi communicate with the Learning locker app (worker,UI,node) .

**Comment by samjaved** [comment]
HttpPost request = new HttpPost(learningLockerUrl);
request.addHeader("content-type", "application/json;charset=UTF-8");
request.addHeader("Authorization", learningClientKey);
request.addHeader("X-Experience-API-Version", "1.0.0");

try {
StringEntity params = new StringEntity(statemnets.toString());
request.setEntity(params);
try {
response = httpClient.execute(request);
System.out.println(response.getStatusLine().getStatusCode());
} catch (ClientProtocolException e) {
// TODO Auto-generated catch block
e.printStackTrace();
} catch (IOException e) {
// TODO Auto-generated catch block
e.printStackTrace();
}
} catch (UnsupportedEncodingException e) {
// TODO Auto-generated catch block
e.printStackTrace();
}
statusCode = response.getStatusLine().getStatusCode();
return statusCode;

Where:
learninglocker.url=http://yourip:8081/data/xAPI/statements
learninglocker.client.key=Basic yourtoken

This is the request that i am sending to learning locker api to send
statements from my system to learning locker .


For more inforamtion that how i am extracting data from learning locker
database.you can take a look at my repo:
https://github.com/samjaved/OpenLAP-xAPI

**Comment by pvHieu3001** [comment]
thank you, this solved my problem :))
